src/fosslight_dependency/_package_manager.py

```
@@ -144,26 +151,44 @@ def run_gradle_task(self):
                             logger.warning(f"Cannot print 'depends on' information. (fail {cmd}: {e})")
 
                     if ret_plugin:
-                        cmd = [cmd_gradle, f':{self.app_name}:generateLicenseTxt']
-                        try:
-                            result = subprocess.run(cmd, capture_output=True, encoding='utf-8')
-                            if result.returncode != 0:
+                        if self.package_manager_name == const.ANDROID:
+                            cmd = [cmd_gradle, f':{self.app_name}:generateLicenseTxt']
+                            try:
+                                result = subprocess.run(cmd, capture_output=True, encoding='utf-8')
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)